### ISSUE TITLE
gh-pages to PDF converter

### ISSUE THREAD

**Issue by Symbian9** [tschaub/gh-pages#155]
Is it a way generate PDF from gh-pages repositary online?
* http://github.com/openorienteering/mapper-manual

**Comment by abdulhannanali** [comment]
@Symbian9 I am not sure, if this is relevant to what this repository does. `gh-pages` pushes the specified folder to `gh-pages` branch by default, it doesn't really make any  assumptions about the content of the repo and doesn't deal with it directly in any way.

**Comment by kylecordes** [comment]
Right, this seems pretty far from the purpose of this package. The package name suggests a strong connection to Github Pages, but actually this is a super useful general purpose tool which happens to be excellent for pushing some assets to a branch so they can be served by Pages. This package doesn't actually do anything with the contents in there, and has nothing to do with PDFs etc.
